index.bs

```diff
@@ -3503,6 +3503,15 @@ the following algorithm returns <i>true</i>.
             <a href="#limit-bigint-numeric-overloading">a separate restriction on their use in
             overloading</a> below. Please also note <a href="#limit-bigint-numeric-unions">the
             advice about using unions of these types</a>.
+        1.  A [=callback function=] that does not have
+            [{{LegacyTreatNonObjectAsNull}}] extended attribute is
+            distinguishable from a dictionary-like category object based on
+            whether the object is [=ECMAScript/callable=]. When selecting a
```

Comment:
```suggestion
        1.  A [=callback function=] that does not have
            [{{LegacyTreatNonObjectAsNull}}] extended attribute is
            distinguishable from a dictionary-like type. When selecting a
```

I think the second sentence clarifies that the Callable algorithm is used, but what's normatively important in this algorithm is that callback functions without LTNOAN are distinguishable from dictionary-like types. (Since this algorithm operates on WebIDL types rather than ECMAScript values, it cannot actually normatively use Callable here.) Separating them with a full stop makes that clearer to me.

As a nit, the normative sentence here should probably discuss the types rather than the objects, for the same reason.

Reply:
I understand that the ECMAScript value is the ting that can be Callable, and it seems that at least the union resolution will convert a Callable ECMAScript value to a callable type. Otherwise, it will convert it to a dictionary-like type?

I'm not sure what you mean about a full stop separating things though.

Reply:
Ok, I've updated this to explicitly separate normative text from non-normative notes. I don't think a punctuation mark is sufficient to imply a switch from normative to non-normative text. 

Let me know if this reads better

Reply:
lgtm thanks